core/src/templates/components/local-footer/local-footer.twig

```diff
@@ -101,7 +101,7 @@
     responsive behaviour so they are not optional. Empty cells will still
     render and create whitespace.
   #}
-  <section class="su-local-footer__columns">
```

Comment:
the closing tag `</section>` will also need to change in line 206

Reply:
@cjwest the closing section tag will need to be changed as well.

Reply:
Good catch!

Reply:
Thanks reviewing this @sherakama